dev-client/src/components/SiteCard.tsx

```diff
@@ -71,15 +66,8 @@ export const SiteCard = ({
       onPress={onCardPress}
       buttons={buttons}
       isPopover={isPopover}>
-      {project && <Heading size="md">{project.name}</Heading>}
-      <Text variant="subtitle2" color="text.secondary">
-        {t('site.last_updated', {
-          date: TEMP_MODIFIED_DATE,
-          name: TEMP_MODIFIED_NAME,
```

Comment:
Issue: I noticed when trying out this change, site cards on the map are now choosing their width based on the length of the title: 
![image](https://github.com/techmatters/terraso-mobile-client/assets/5590815/b17561fd-ad50-4ea8-b8f3-f45d6d8ddb4c)

I imagine this was a problem before, but because we had the extra date and name text it wouldn't be as noticeable. Not immediately sure what to recommend though.

Reply:
Fixed:
<img width="380" alt="image" src="https://github.com/techmatters/terraso-mobile-client/assets/86784/335b980d-33e3-428e-a596-8b124d4503d5">